Automattic/wp-calypso
# HelpCenter: ensure closed state on unmount

#### Proposed Changes

* This PR moves help center callback logic from checkout to a hook
* Modify help center container to call `onClose` on unmount

#### Testing Instructions
* Checkout this branch
* Run `yarn start`
* Open the checkout page for a simple site
* Open the help center
* Exit the checkout page by clicking the x button on the top left
* Visit the checkout page again
* Ensure that the initial state of the help center is closed instead of open

Related to #65930
